spec/requests/api/reservations_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'swagger_helper'
+
+RSpec.describe 'api/reservations', type: :request do
+  path '/api/users/{user_id}/reservations' do
```

Comment:
Please change the path to reservations to `'api/reservations'`.
The `user_id` in the path is a query parameter and not a path parameter.